CHANGES_NEXT_RELEASE

```diff
@@ -11,4 +11,4 @@
 - Fix: GET /v2/entities/id/attrs/name/value for type date (Issue #1885)
 - Fix: PUT /v2/entities/id/attrs/name/value forbidden chars (JSON) (Issue #1905)
 - Fix: PUT /v2/entities/id/attrs/name/value changing type to 'none' (text/plain) (Issue #1904)
-
+- Add: PUT /admin/log?level=XXX to change log level in run-time (Issue #1913)
```

Comment:
Beyond changes in the code, this PR should include changes in the documentation. In particular:

* Changes in doc/manuals/admin/management_api.md describing the new API operation (probably in a new subsection). Copy-pasting from the #1913 could help.
* Changes in doc/manuals/admin/logs.md#log-format, bullet "lvl (level)" mentioning that log level can be changed dynamically (including a link to the aforementioned section regarding how to do it)
* Changes in doc/manuals/admin/cli.md, `-logLevel flag` explaining that this is *the initial* log level at startup time and that it can be changed dynamically (including a link to the aforementioned section regarding how to do it). Probably some changes in the -u and -U messages regarding logLevel should be done in the same line.